Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
```

Comment:
@mehdisadeghi not sure if this holds, but are you not able to use the python image? I recently used a ruby image and installed nodejs on top using the same commands that you have below. Although I couldnt get the node image to accept Ruby so I am not entirely sure about what I am saying ....

https://hub.docker.com/_/python/

Reply:
@carocad I used that in the beginning then I realized the image is too big so I swapped it out for Ubuntu Bionic which was 85MB large.

Reply:
@mehdisadeghi I just recently learn that there are some images called `alpine`. Those images are optimized for file-size. That's how I got the `kamal` size so small.

Python also has an `alpine` version which you could use to minimize the size; according to the README, the size is like 5 MB, which is pretty much nothing

Reply:
@carocad  yes they are awesome! I tried first with alpine but was unable to install pyosmium. I think I had to install a lot of stuff to make it work so I switched to Ubuntu

Reply:
Oh I see. I get it now. Well as we said, this is just a nice to have so we can slowly check better ways to implement it.